### DISCUSSION TITLE
Notebook palettes and Vanilla theme

### DISCUSSION THREAD

**Discussion by kookma** [NicolasPetton/Notebook#29] [Q&A] [answered]
I use the Notebook theme plus its beautiful palettes. Sometime I use other theme specially the vanilla with Notebook palettes. The minor issue is the link on sidebars (like table of contents) are white on hover which has a very low contrast.

Is it possible to fix this color contrast?

**Comment by NicolasPetton** [comment]
> The minor issue is the link on sidebars (like table of contents) are white on hover which has a very low contrast.

Are you referring to the light palettes (purple & blue), or the dark palette?

**Reply by kookma** [reply]
I mean the light palettes (grey and beige)

**Comment by NicolasPetton** [comment]
I made an issue here: https://github.com/NicolasPetton/Notebook/issues/31

**Comment by NicolasPetton** [comment, marked as answer]
The issue is fixed in df5302e, I'll make a patch release soon!
